### PR TITLE
#266 Handle ClassCastException in signature.init calls

### PR DESCRIPTION
**Pull request by p-barabas**

### CODE REVIEW

File: local.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```
Comment (vanitasvitae) [inline local.properties]: Please remove this file from the commit.

File: pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java [hunk 2]
```diff
@@ -134,7 +134,7 @@ protected boolean accept(String userId) {
      * Note: This only accepts user-ids which properly have the email address surrounded by angle brackets.
      *
      * The argument <pre>email</pre> can both be a plain email address (<pre>"foo@bar.baz"</pre>),
-     * or surrounded by angle brackets (<pre>"<foo@bar.baz>"</pre>, the result of the filter will be the same.
```
Comment (vanitasvitae) [inline pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java]: Please revert this change.

File: pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java [hunk 3]
```diff
@@ -181,7 +181,7 @@ protected boolean accept(String userId) {
     }
 
     /**
-     * Filter that filters for user-ids which pass at least one of the given <pre>filters</pre>>.
```
Comment (vanitasvitae) [inline pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java:184]: Good catch. Can you put this into its own commit?

**Review by vanitasvitae — CHANGES_REQUESTED** [review]

**Comment by vanitasvitae** [comment]
I created #268 which prevents the library from attempting to verify signatures using the wrong key in the first place.
I think this together with your changes should definitely fix the issue.

File: local.properties (hunk 1 above)
Comment (p-barabas) [reply local.properties]: Done.

File: pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java (hunk 2 above)
Comment (p-barabas) [reply pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java]: Done.

File: pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java (hunk 3 above)
Comment (p-barabas) [reply pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java:184]: Done.

**Comment by vanitasvitae** [comment]
Applied as 8c6813ce565567ed84d5b1e0a5ee32a14634a984
